Formula/ivykis.rb

```diff
@@ -0,0 +1,31 @@
+class Ivykis < Formula
+  desc "Async I/O-assisting library"
+  homepage "http://libivykis.sourceforge.net"
+  url "https://github.com/buytenh/ivykis/archive/v0.41.tar.gz"
```

Comment:
Let's fetch it from here instead: https://sourceforge.net/projects/libivykis/files/0.41/